### PR TITLE
Fixes #1193. Create a simple /issues/new view for reporting bugs

### LINKED ISSUE #1193: Make /issues/new endpoint a reduced version of the homepage reporting form
If we do this correctly, this could fix #1163, #1164, #1165. This will also fix #317.

Currently we POST form data to that endpoint, this bug should create a response for GET, which is the form, a reduced header, and not much else (if anything).

### PR DESCRIPTION
**Pull request by miketaylr**
I need to write some tests, not ready for review yet.

### CODE REVIEW

File: webcompat/views.py [hunk 1]
```diff
@@ -155,8 +155,22 @@ def show_issues():
...
-@app.route('/issues/new', methods=['POST'])
+@app.route('/issues/new', methods=['GET', 'POST'])
 def create_issue():
+    """Creates a new issue
+
+    GET will return an HTML response for reporting issues
+    POST will create a new issue
+    """
+    if request.method == 'GET':
+        bug_form = IssueForm()
```
Comment (miketaylr) [inline webcompat/views.py]: This starts to feel a bit redundant w/ the index view, but I'm not sure how to make it more DRY. 🍤

**Comment by miketaylr** [comment]
(bunch of random Travis failures right now... possibly GitHub issues?)

File: webcompat/views.py (hunk 1 above)
Comment (karlcow) [inline webcompat/views.py]: What about making the redundant code into a function that you call in both places?

**Comment by miketaylr** [comment]
> (bunch of random Travis failures right now... possibly GitHub issues?)

Yeah, the issue was a class was changed and the logout helper method was failing. Fixed now.

r? @karlcow 
r? @magsout (for CSS stuff)

File: webcompat/views.py [hunk 2]
```diff
@@ -155,8 +150,18 @@ def show_issues():
     return render_template('list-issue.html', categories=categories)
 
 
-@app.route('/issues/new', methods=['POST'])
+@app.route('/issues/new', methods=['GET', 'POST'])
 def create_issue():
+    """Creates a new issue
```
Comment (karlcow) [inline webcompat/views.py]: nitpicking: Missing '.' at the end of the sentences.

File: webcompat/views.py [hunk 3]
```diff
@@ -155,8 +150,18 @@ def show_issues():
...
+
+    GET will return an HTML response for reporting issues
+    POST will create a new issue
+    """
```
Comment (karlcow) [inline webcompat/views.py]: I have the feeling we could improve the way we handle the code here. But probably for another issue, you might want to open.
Right now we have

```python
if GET:
    this_code()
    return blah_get
# here the http post code (with logging and spam)
return blah_post
```

Maybe something along

```python
# common_tasks
# ua_header, etc.
if GET:
    this_code()
    return blah_get
if POST:
   # making functions for different features
    spam_protect()
    log_this()
    return blah_post
```

BUT Let's not block the progress of this PR. so +1 with the issue created.

**Comment by magsout** [comment]
r+ @miketaylr 

Good job

**Comment by miketaylr** [comment]
Thanks @magsout!

**Comment by miketaylr** [comment]
Also, thanks @karlcow! https://github.com/webcompat/webcompat.com/issues/1200 filed for refactoring of create_issue method.

**Comment by miketaylr** [comment]
Fixed up the docstring nit. Will merge once tests are green and cut a new release/deploy.

**Pull request merged**